src/components/views/ScreeningsSection.vue

```diff
@@ -12,16 +12,23 @@ import movies from '@/stores/moviesStore'
 
 export default defineComponent({
   components: { SectionTitle, SectionSubtitle, AppLabel, AppButton, AppSelect, ScreeningsList },
+  props: {
+    singleMovie: {
+      type: Array,
+      default: null
+    }
```

Comment:
This seems... weird 😅 
Shouldn't this have been a Boolean with a default of `false`? 👀

Reply:
This is actually an array of one element that contains the data of a single movie to be displayed on the screening card on the SingleMoviePage (maybe I should name it "singleMovieData" then?) I could have created a separate bolean value to inform ScreeningsSection if it's handling the case of a single movie page or general screenings list, but checking if singleMovie array exists already do the job of such boolean. Shall I change it to make the code easier to read and understand?

Reply:
Ah, now I see you then pass this data onto the screenings section.
Well if it's going to always be one movie, then an Object would be a better type and we could just pass it wrapped in the array there in the template where it is *actually* needed 😄